pkg/chartutil/create.go

```diff
@@ -511,13 +511,12 @@ spec:
 var Stderr io.Writer = os.Stderr
 
 // CreateFrom creates a new chart, but scaffolds it from the src chart.
-func CreateFrom(chartfile *chart.Metadata, dest, src string) error {
+func CreateFrom(chartname, dest, src string) error {
```

Comment:
So unfortunately, this would constitute a breaking change as chartutil.CreateFrom is a publicly exposed function call.  
see this: https://pkg.go.dev/helm.sh/helm/v3@v3.6.3/pkg/chartutil#CreateFrom